
Property changes on: src/network/core
___________________________________________________________________
Added: svn:ignore
   + .tcp_admin.h.swp


Index: src/network/core/tcp_admin.h
===================================================================
--- src/network/core/tcp_admin.h	(revision 24376)
+++ src/network/core/tcp_admin.h	(working copy)
@@ -360,6 +360,7 @@
 	 * uint64  Company value (previous quarter).
 	 * uint16  Performance (previous quarter).
 	 * uint16  Delivered cargo (previous quarter).
+	 * uint64  Company value (realtime value).
 	 * @param p The packet that was just received.
 	 * @return The state the network should have.
 	 */
Index: src/network/network_admin.cpp
===================================================================
--- src/network/network_admin.cpp	(revision 24376)
+++ src/network/network_admin.cpp	(working copy)
@@ -421,6 +421,8 @@
 			p->Send_uint16(min(UINT16_MAX, company->old_economy[i].delivered_cargo.GetSum<OverflowSafeInt64>()));
 		}
 
+		p->Send_uint64((int64)CalculateCompanyValue(company));
+
 		this->SendPacket(p);
 	}
 
